tests/emukit/bayesian_optimization/test_entropy_search.py

```diff
@@ -55,3 +66,12 @@ def test_innovations(acquisition):
     dm, dv = acquisition._innovations(x)
     assert np.any(np.abs(dm) > 1e-3)
     assert np.any(np.abs(dv) > 1e-3)
+
+
+def test_multi_information_source_entropy_search_shape(multi_source_acquisition):
+    rng = np.random.RandomState(43)
+
+    x_test = np.concatenate([rng.rand(2, 1), np.array([[0], [1]])], axis=1)
```

Comment:
Fixing random seed and calling random is just as good as hardcoding numbers. The latter however is simpler.

That is, unless random is also called somewhere down the line, but does not look like there is any

Reply:
Done